sdk/synapse/Azure.Analytics.Synapse.Spark/tests/SparkBatchClientLiveTests.cs

```diff
@@ -47,7 +46,34 @@ public async Task TestSparkBatchJob()
             SparkBatchOperation createOperation = await client.StartCreateSparkBatchJobAsync(createParams);
             SparkBatchJob jobCreateResponse = await createOperation.WaitForCompletionAsync();
 
-            // Verify the Spark batch job completes successfully
+            // Verify the Spark batch job submission completes successfully
+            Assert.True("starting".Equals(jobCreateResponse.State, StringComparison.OrdinalIgnoreCase) || "running".Equals(jobCreateResponse.State, StringComparison.OrdinalIgnoreCase),
+                string.Format(
+                    "Job: {0} did not return success. Current job state: {1}. Error (if any): {2}",
+                    jobCreateResponse.Id,
+                    jobCreateResponse.Result,
+                    string.Join(", ", jobCreateResponse.Errors ?? new List<SparkServiceError>())
+                )
+            );
+
+            // Get the list of Spark batch jobs and check that the submitted job exists
+            List<SparkBatchJob> listJobResponse = await SparkTestUtilities.ListSparkBatchJobsAsync(client);
+            Assert.NotNull(listJobResponse);
+            Assert.IsTrue(listJobResponse.Any(job => job.Id == jobCreateResponse.Id));
+        }
+
+        [RecordedTest]
+        public async Task TestSparkBatchJobExecution()
```

Comment:
Could we name this test something a little more descriptive?  For example, something like `SparkBatchJobCompletesWhenJobComplete`, (and possibly rename the first one to `SparkBatchJobCompletesWhenJobStarts` ?)

Reply:
Sure. I will fix this soon

Reply:
Fixed